app/items.py

```diff
@@ -0,0 +1,18 @@
+class Armour:
+    def __init__(self, armour_conf: dict) -> None:
+        self.part = armour_conf["part"]
+        self.protection = armour_conf["protection"]
+
+
+class Potion:
+    def __init__(self, potion_conf: dict) -> None:
+        self.name = potion_conf["name"]
+        self.protection_effect = potion_conf["effect"].get("protection")
+        self.power_effect = potion_conf["effect"].get("power")
+        self.hp_effect = potion_conf["effect"].get("hp")
```

Comment:
Useless class. Code is so hard due to this. Because If we will have 100 effects???